lib/mnit/input.nit

```diff
@@ -34,7 +35,12 @@ interface PointerEvent
 
 	# Is down? either going down or already down
 	fun pressed: Bool is abstract
+
+	# Is it not currently pressed down?
 	fun depressed: Bool is abstract
```

Comment:
you should document the relation with `pressed`; by the way, why the default implementation is not `return not pressed`?